react-hot-button-component/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "react-hot-button-component",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "build":"webpack",
+    "watch":"webpack --watch",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "@babel/core": "^7.13.10",
+    "@babel/plugin-transform-react-jsx": "^7.12.17",
+    "babel-loader": "^8.2.2",
+    "react": "^17.0.1",
+    "react-dom": "^17.0.1",
+    "webpack": "^5.26.3",
+    "webpack-cli": "^4.5.0"
```

Comment:
Reminder: everything but `react` and `react-dom` should be saved as `devDependencies`